rkt/pods.go

```diff
@@ -733,14 +734,33 @@ func (p *pod) getState() string {
 
 // getPID returns the pid of the pod.
 func (p *pod) getPID() (pid int, err error) {
+	// check the "pid" (process ID) file if it exists, and on "ppid"
+	// (parent process ID). Recent stage1 uses "ppid" but we still
+	// support older rkt versions or alternative stage1 implementations.
```

Comment:
I am thinking of reverting a part the patch in #826 to read the pid files in the `enter` entrypoint rather than in stage0. My idea is that the pid is an implementation detail of stage1 and should not leak in stage0: other stage1 such as ones based on KVM might not have a process leader for the container. In this way, the pid files would not need to be specified in the [Stage1 ACI implementor's guide](https://github.com/coreos/rkt/blob/master/Documentation/devel/stage1-implementors-guide.md#rkt-enter--coreoscomrktstage1enter).